giraffe/src/types/index.ts

```diff
@@ -453,6 +455,12 @@ export interface ScatterLayerConfig {
   symbol?: string[]
 }
 
+export interface SimpleTableLayerConfig {
+  type: 'simple table' // do not refactor or restrict to LayerTypes.SimpleTable
+  showAll: boolean
+  fluxResponse?: string
+}
+
```

Comment:
Same as above. Not a big deal but this should have waited until there was code that uses this definition.